Sources/Testing/ABI/Encoded/ABI.EncodedIssue.swift

```diff
@@ -30,8 +30,17 @@ extension ABI {
     var _severity: Severity
 
     /// Whether or not this issue is known to occur.
+    ///
+    /// This should perhaps be deprecated in favor of `_knownIssueContext`.
     var isKnown: Bool
 
+    /// An ``Issue/KnownIssueContext-swift.struct`` representing the
+    /// ``withKnownIssue(_:isIntermittent:sourceLocation:_:when:matching:)`` call
+    /// that matched this issue, if any.
+    ///
+    /// - Warning: Severity is not yet part of the JSON schema.
```

Comment:
This comment about "Severity" looks copy/pasted from elsewhere but isn't relevant

Reply:
Fixed.